docs/getting-started-guide.md

```diff
@@ -0,0 +1,168 @@
+# How to set up and run the Co-Simulation Demo
+
+This demonstration shows how to compile and run the Co-Simulation demo of Buildroot in QEMU with a simulated device in SystemC. This configeration is tested working for Ubuntu 18.0.4 and assumes that `cosim` is located in your home directory. This walkthough also assumes the device being emulated is the Zynq-7000 SoC.
+
+## Dependencies
+Below are the dependencies needed to compile all the libraries in this demo:
+```bash
+sudo apt update
+sudo apt install cmake gmake gcc qemu-kvm qemu-system qemu-user-static verilator
+```
+
+## Setup and Compilation
+Run these commands to clone and build the necessary repos (`~/cosim` assumed as the base directory).
+
+### Create the base directory
+```bash
+mkdir ~/cosim
+```
+
+### SystemC Setup
+```bash
+cd ~/cosim
+SYSC_VERSION=systemc-2.3.2
+wget https://www.accellera.org/images/downloads/standards/systemc/systemc-2.3.2.tar.gz
+tar xf ${SYSC_VERSION}.tar.gz && cd ${SYSC_VERSION}/
+
+# Build and install the binaries
+CXXFLAGS=-std=c++11 ./configure ; make
+sudo make install
```

Comment:
Perhaps not using the the SYSC_VERSION and moving the comment to the top is more clear:

```
# Build and install the binaries
$ cd ~/cosim
$ wget https://www.accellera.org/images/downloads/standards/systemc/systemc-2.3.2.tar.gz
$ tar xf systemc-2.3.2.tar.gz
$ cd systemc-2.3.2/
$ CXXFLAGS=-std=c++11 ./configure --prefix=/usr/local/systemc-2.3.2
$ make
$ sudo make install
```